### PR TITLE
add manifest v8 compatibility

### LINKED ISSUE #24: Support v8 manifest
Hey all,

I noticed that there's a new manifest version. Happy to help with this if no one is working on it yet

https://schemas.getdbt.com/dbt/manifest/v8.json

### PR DESCRIPTION
**Pull request by dlawin**
Closes #24

### CODE REVIEW

**Comment by dlawin** [comment]
Should i generate artifacts for jaffleshop? Wasn't sure how those were being created

**Comment by yu-iskw** [comment]
@dlawin Thank you for the contribution. I would like to add some unit tests, no matter how simple is. The jaffle_shop project would be good for unit tests at the modment.

**Comment by dlawin** [comment]
> @dlawin Thank you for the contribution. I would like to add some unit tests, no matter how simple is. The jaffle_shop project would be good for unit tests at the modment.

added

File: dbt_artifacts_parser/parser.py [hunk 1]
```diff
@@ -15,7 +15,6 @@
 #  limitations under the License.
 #
 from typing import Union
-from dbt_artifacts_parser.parsers.manifest.manifest_v7 import ManifestV7
```
Comment (yu-iskw) [inline dbt_artifacts_parser/parser.py:18]: Thank you for reordering imports.

**Review by yu-iskw — APPROVED** [review]

**Comment by yu-iskw** [comment]
@dlawin Thank you for updating the parser together. I have merged it into the default branch.

**Comment by yu-iskw** [comment]
We have published v0.2.4. Can you give it a try on your use case?
https://github.com/yu-iskw/dbt-artifacts-parser/releases/tag/v0.2.4

**Comment by dlawin** [comment]
> We have published v0.2.4. Can you give it a try on your use case? https://github.com/yu-iskw/dbt-artifacts-parser/releases/tag/v0.2.4

Yes I can test in the next day or two!

**Comment by dlawin** [comment]
> We have published v0.2.4. Can you give it a try on your use case? https://github.com/yu-iskw/dbt-artifacts-parser/releases/tag/v0.2.4

works great

**Pull request merged**